### PR TITLE
Converter sec aif cif

### PR DESCRIPTION
**Pull request by phuang26**

### CODE REVIEW

File: converter_app/readers/sec.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        return result
+
+    def _append_table(self, tables):
+        self._has_header = False
+        self._has_first_value = False
+        self._is_table_empty = True
+
+        return self.append_table(tables)
+
+    def _handle_line(self, line: str, table: dict[str, list | dict]):
```
Comment (jochenklar) [inline converter_app/readers/sec.py:36]: These type hints will seem to only work in Python >= 3.10 which is pretty new (for Python standards). Dropping Python <= 3.9 needs to be decided before merging this PR.

**Comment by jochenklar** [comment]
There is an error in the `SecReader` (I cannot easily locate in the code): The Table #6 from the example file should have 7 columns, but it has 10. This is because `Integral[%]	PSS SLD2020/9000` is split into `Integral[%]` and `PSS SLD2020/9000` while it should be one column. The columns are seperated by tabs.

File: converter_app/readers/sec.py (hunk 1 above)
Comment (jochenklar) [reply converter_app/readers/sec.py:36]: Can be fixed with adding `from __future__ import annotations` at the very top of the file. No need for a Python 3.10 discussion.

**Comment by StarmanMartin** [comment]
I removed Type annotation
fixed integral split

**Comment by phuang26** [comment]
@StarmanMartin can you please provide the sample files and the profiles from your local server for further testing?

you can upload the files or send it by email to Jochen and me, thank you.

**Comment by StarmanMartin** [comment]
A few files for testing:

AIF: 
[cm1c01804_si_003.txt](https://github.com/ComPlat/chemotion-converter-app/files/10138654/cm1c01804_si_003.txt)
[cm1c01804_si_004.txt](https://github.com/ComPlat/chemotion-converter-app/files/10138655/cm1c01804_si_004.txt)

CIF:
[SG-V3545-6-13.cif.zip](https://github.com/ComPlat/chemotion-converter-app/files/10138656/SG-V3545-6-13.cif.zip)

SEC:
[MS-185_Lichtstreuung.TXT](https://github.com/ComPlat/chemotion-converter-app/files/10138672/MS-185_Lichtstreuung.TXT)

**Pull request merged**